### ISSUE TITLE
Check the general.material use before release

### ISSUE THREAD

**Issue by Bertram25** [OpenDungeons/OpenDungeons#561]
I can't remember whether this one is still actually useful.

**Comment by akien-mga** [comment]
I moved it away from the materials/scripts folder locally and did not notice any obvious regression, but then that's a bit light to know if it's useful or not ;-)

Using my magic "ack" command, I found nothing relevant about "general" in the code (it's important to use `ack general` and not `ack -i general`, because with the latter you get all our headers citing the GPL :-P).

**Comment by Bertram25** [comment]
@akien-mga In the general.material file, the material is named base_material.

Could you also ack on this one? (I'll have a look as well if you can't for any reason.)

**Comment by akien-mga** [comment]
> In the general.material file, the material is named base_material.
> Could you also ack on this one? (I'll have a look as well if you can't for any reason.)

That's correct. Here is the output:

```
$ ack -i base_material
materials/scripts/general.material
1:material base_material
```

IMO this general.material is just a template. It should probably be renamed to template.material to avoid confusion (and we should check if it's still relevant at all, or if it is outdated).

**Comment by Bertram25** [comment]
> IMO this general.material is just a template. It should probably be renamed to template.material to avoid confusion (and we should check if it's still relevant at all, or if it is outdated).

In fact, it's a remnant from the past when someone (?) tried to play with material files to factorize their content using the import, override parameter mechanism.
It didn't turn well as when I made the game run the first time, I spent one afternoon fixing all the modified materials back at the time. Not blaming anyone in particular. A lot of things were messy at the time.

If it is not used, it should be removed as it is plainly obsolete.
